rust/scripts/build-release.sh

```diff
@@ -75,7 +80,7 @@ main() {
     # generate filcrypto.h
     # Check if FVM is in the build features - if so, include it in header generation
     local __header_features="c-headers"
-    if echo "${@:2}" | grep -q "fvm"; then
+    if echo "${@:2}" | tr ' ,' '\n' | grep -qx "fvm"; then
```

Comment:
```suggestion
    if [[ -z "${FFI_DISABLE_FVM}" ]]; then
```

this is my remaining suggestion to get parity with install-filcrypto, then they both use `FFI_DISABLE_FVM` only and don't rely on passed arguments - cause we still don't pass `fvm` via the GHA config so this wouldn't pick it up for header generation - so we have the library built with fvm, but filcrypto.h missing the fvm function declarations.